src/interp/binary-reader-interp.cc

```diff
@@ -815,6 +826,12 @@ wabt::Result BinaryReaderInterp::OnImportTable(Index import_index,
   CHECK_RESULT(CheckImportKind(module_name, field_name, ExternalKind::Table, export_->kind));
 
   Table* table = env_->GetTable(export_->index);
+  if (elem_type != table->elem_type) {
```

Comment:
does this need to be a subtype relationship?

Reply:
I'm not sure.  At least right now none of the tests expect that.  It could be similar to the imported global case where when the import is mutable it has to be an exact match.. and tables are always mutable right now.